src/page/blog/articles-page.jsx

```diff
@@ -1,10 +1,13 @@
 import { useQuery } from "@apollo/client";
+import { useContext } from "react";
 import { Footer, Post } from "../../components/index.js";
 import { GET_BLOG_POSTS } from "../../graphql/queries/blog/posts.js";
 import purpleGoblin from "../../images/purple-goblin.png";
+import { CurrentUserContext } from "../../store/current-user-context.js";
 import { formatDateTime } from "../../utils/constants.js";
 
 export const Articles = () => {
+  const { currentUser } = useContext(CurrentUserContext);
```

Comment:
Nitpick: you could've imported useCurrentUser. Then this line could've been written as

`const { currentUser } = useCurrentUser();`

Reply:
I actually think this is more than a 'nit'. @ATribbey since we already have the `useCurrentUser` hook, let's leverage that here instead of reinventing the wheel a bit. That way we also won't need to import `useContext`.